google-youtube-upload.html

```diff
@@ -393,6 +422,30 @@
         value: Boolean('MediaRecorder' in window &&
                        'mediaDevices' in navigator &&
                        'getUserMedia' in navigator.mediaDevices)
+      },
+
+      /**
+       * The [MediaStream audio constraints](https://developer.mozilla.org/docs/Web/API/MediaStreamConstraints/audio).
+       */
+      audio: {
+        type: Object,
```

Comment:
For a `Boolean` property to be configurable from markup, it must default to `false`. If it defaults to `true`, it is not possible to set it to `false` from markup, since the presence of the attribute, with or without a value, equates to `true`. This is the standard behavior for attributes in the web platform.

Since this behavior doesn't fit this use case (video/audio track should be included in the stream in case the `video` / `audio` attribute is not presented in element markup — this is how it works now, the user just write `<google-youtube-upload client-id="..."></google-youtube-upload>` instead of `<google-youtube-upload video audio client-id="..."></google-youtube-upload>`, and audio and video tracks are included in the steram by default; if we change this behavior, it will be a breaking change), I use an `Object` instead  `Boolean`, because `true` and `false` are valid values (subset) of `Object`.